src/codeu/chat/client/ClientMessage.java

```diff
@@ -14,16 +14,9 @@
 
 package codeu.chat.client;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-import codeu.chat.common.Conversation;
-import codeu.chat.common.ConversationSummary;
-import codeu.chat.common.Message;
-import codeu.chat.common.Uuid;
-import codeu.chat.common.Uuids;
+import java.util.*;
```

Comment:
Revert back to old imports. In general, it's not a good idea to do imports with .* at the end